tests/unit/model/grids/test_grid_search.py

```diff
@@ -13,7 +13,7 @@
 
 def test_grid_get_nearest_substation_node(basic_grid):
     substation_node = basic_grid.get_nearest_substation_node(node_id=103)
-    assert NodeType.SUBSTATION_NODE == NodeType(substation_node.node_type)
+    assert NodeType.SUBSTATION_NODE == NodeType(int(substation_node.node_type))
```

Comment:
```suggestion
    assert NodeType.SUBSTATION_NODE == NodeType(substation_node.node_type.item())
```
I'd prefer this version. More consistent with the rest of the project.